### PR TITLE
Add packages informations from c3pm.yaml

### PR DESCRIPTION
**Pull request by MinaBarry**
(back + front, waiting for the fix of the CLI pr.)

### CODE REVIEW

**Comment by chloebourbion** [comment]
@MinaBarry can you fix the test so we can merge?

**Review by angela-hubner — DISMISSED** [review]

File: front/components/pages/packages/MarkdownDisplayer/index.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ JSX.Element {
   const classes = useStyles();
 
   return (
-    <div className={classes.readme}>
+    <div className={classes.readme} id="readme-displayer">
```
Comment (ItagibaALIX) [inline front/components/pages/packages/MarkdownDisplayer/index.tsx]: Why did you add an "id="readme-displayer" ?

Comment (chloebourbion) [reply front/components/pages/packages/MarkdownDisplayer/index.tsx]:
```suggestion
    <div className={classes.readme}>
```

**Review by ItagibaALIX — APPROVED** [review]
